plugins/catalog-backend/src/ingestion/processors/AwsOrganizationProcessor.ts

```diff
@@ -0,0 +1,131 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ComponentEntityV1alpha1,
+  Entity,
+  LocationSpec,
+} from '@backstage/catalog-model';
+import AWS, { Organizations } from 'aws-sdk';
+import { Account, ListAccountsResponse } from 'aws-sdk/clients/organizations';
+
+import * as results from './results';
+import { CatalogProcessor, CatalogProcessorEmit } from './types';
+
+const AWS_ORGANIZATION_REGION = 'us-east-1';
+const LOCATION_TYPE = 'aws-organization';
+
+export class AwsOrganizationProcessor implements CatalogProcessor {
```

Comment:
The other processors named something with "org" import users and groups, but this one imports cloud accounts. Hm. But they are called `*OrgReader*`. Naming is hard, OK, nevermind :)

Reply:
https://twitter.com/codinghorror/status/506010907021828096

Reply:
Yes, naming is hard - maybe something like `AwsOrganizationCloudAccountProcessor` is more specific? I could see folks getting confused if they have some type of organizational data stored in some AWS service

Reply:
Sounds good.